restful_tango/tangoREST.py

```diff
@@ -111,16 +111,19 @@ def checkFileExists(self, directory, filename, fileMD5):
                 except IOError:
                     continue
 
-    def createTangoMachine(
-        self, image, vmms=Config.VMMS_NAME, vmObj={"cores": 1, "memory": 512}
-    ):
+    def createTangoMachine(self, image, vmms=Config.VMMS_NAME, vmObj=None):
         """createTangoMachine - Creates a tango machine object from image"""
+        cores = Config.DOCKER_CORES_LIMIT
+        memory = Config.DOCKER_MEMORY_LIMIT
```

Comment:
These two lines should use `getattr` so that existing installations don't break when their Config object does not define the limits

<img width="773" alt="Screenshot 2023-12-30 at 02 40 36" src="https://github.com/autolab/Tango/assets/9074856/ee7093ca-c81e-44fb-aa5a-7199f29f1f22">

e.g.
```
cores = getattr(Config, "DOCKER_CORES_LIMIT", None)
memory = getattr(Config, "DOCKER_MEMORY_LIMIT", None)
```